src/osp/Universe.h

```diff
@@ -69,7 +69,7 @@ class Universe
 
     using Reg_t = entt::basic_registry<Satellite>;
 
-    Universe();
+    Universe() = default;
     Universe(Universe const &copy) = delete;
     Universe(Universe &&move) = delete;
```

Comment:
recommend deleting the operator= functions too, for consistency.

Reply:
I like to define a macro OSP_DISABLE_COPY_AND_MOVE(Universe) that just deletes all of em.